lib/phoenix_live_view/test/live_view_test.ex

```diff
@@ -1574,7 +1574,13 @@ defmodule Phoenix.LiveViewTest do
 
         {:unix, _} ->
           if System.find_executable("cmd.exe") do
-            {"cmd.exe", ["/c", "start", path]}
+            win_path_regex = ~r/^([^\\]+\\)*?$/
```

Comment:
Could we check instead if the path has a `\\`? Or is it important to check that it does not start with a `\\`?